.github/workflows/release.yml

```diff
@@ -308,6 +308,20 @@ jobs:
         env:
           COMMITTER_TOKEN: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
 
+  winget_update:
+    name: Update Winget Manifest
+    runs-on: windows-latest
+    needs: [release_please, github_build, upload_artifacts]
+    if: ${{ needs.release_please.outputs.release_created == 'true' }}
+    env:
+      URL_64: https://github.com/starship/starship/releases/download/${{ needs.release_please.outputs.tag_name }}/starship-x86_64-pc-windows-msvc.msi
+      URL_32: https://github.com/starship/starship/releases/download/${{ needs.release_please.outputs.tag_name }}/starship-i686-pc-windows-msvc.msi
+    steps:
+      - run: |
+          $version = '${{ needs.release_please.outputs.tag_name }}'.replace('v', '')
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
```

Comment:
This seems to be an official install method, but if possible, I would prefer using `winget install` for this.

Reply:
I think the reason this method is considered official is because winget isn't supported on Windows Server (per their README) which this action will run on: https://github.com/microsoft/winget-cli/blob/master/README.md